aquifer/aquifer
# D8 Create and Build

Built off of @patrickocoffeyo's D8 branch and translated some of the changes to the new standard. Also, Fixed up the src files added by Patrick to actually work with D8.
## Testing

To Test, you can build a new d8 site by doing `aquifer projectname -d 8` then do an `aquifer build` inside the project.

If you haven't tested a stand alone version of aquifer before... do the following:
- clone aquifer to a directory `git clone https://github.com/aquifer/aquifer.git`
- add my fork as a remote `git remote add generalredneck https://github.com/generalredneck/aquifer.git`
- check out the **d8** branch. `git fetch generalredneck; git checkout generalredneck/d8`
- run `npm install`
- navigate to where you want to build a project.
- run `/PATH_TO_AQUIFER_CLONE/bin/aquifer.js projectname -d 8`
- `cd projectname`
- `/PATH_TO_AQUIFER_CLONE/bin/aquifer.js build`

**Expected Results**:
You should have a fully built out d8 site that if you set up `settings/secret.settings.php` with a database and run from a web server, should have a fully operational d8 site. The standard module, themes, and root folders should point to the respective points in d8.
